packages/core/src/stories/Installation/javascript.stories.ts

```diff
@@ -0,0 +1,103 @@
+import type { Meta } from '@storybook/html';
+import hljs from 'highlight.js';
+import '../../../.storybook/tegel.syntax.highlighter.css';
+
+const meta: Meta = {
+  title: 'Intro/Installation',
+  parameters: {
+    layout: 'fullscreen',
+    options: {
+      showPanel: false,
+      showToolbar: false,
+    },
+  },
+};
+
+hljs.highlightAll();
+
+export default meta;
+export const Javascript = {
+  render: () => `
+    <style>
+    article {
+        box-sizing: border-box;
+        max-width: 688px;
+
+        padding: 32px;
+        margin: auto;
+        >* {
+            margin-bottom: 72px;
+        }
+    }
+
+    code {
+        border-radius: 4px;
+    }
+</style>
+<article class="tds-u-p2 tds-body-01">
+    <section>
+        <p class="tds-body-01"><strong>Published: 2023-10-11</strong></p>
+        <h1 class="tds-headline-02">Installing Tegel</h1>
+        <p>This is a getting started guide aimed at developers that want to install and use the @scania/tegel library (TDS).
+            This library consists of web components built using Stencil, which means that they are framework agnostic and follow
+           browser standards. The library is installed and updated via NPM (node package manager). For further installation
+            information please see the framework specific installation guides below.
+        </p>
+    </section>
+    
+    <section>
+        <h4>Migration from SDDS</h4>
+        <p>TDS and @scania/components (SDDS) share a lot of design, but the tech implementation have two different approaches.
+        Therefore there is some migration work needed when going from SDDS to TDS. We have outlined all the differences in this
+        <tds-link><a href="?path=/story/intro-migrating-from-components-v4--page">migration document</a></tds-link>. However, since they exist in different namespaces (SDDS is prefixed with ‘sdds’ and TDS with ‘tds’)
+        the two solutions can exist alongside each other, enabling a soft migration that can be done over time.</p>
+    </section>
+
+    <section>
+        <h4>Framework integrations</h4>
+        <p>TDS is, like previously mentioned, a collection of web components. This enables them to work within any frontend
```

Comment:
Should we maybe use the term "framework agnostic"?

Reply:
We do and it once in the introduction, I'm thinking it could be good to have one explanation that is a bit more verbose. Mainly to make sure everyone understands, no matter what english skills they might have. What do you think?